app/models/mdm/web_page.rb

```diff
@@ -1,4 +1,5 @@
 # Web page requested from a {#web_site}.
+require 'webrick'
```

Comment:
@limhoff-r7 maybe i can get your feedback on this one. The the requiring of the Webrick::Cookie was necessary to fix the bug with export. Not sure why Webrick is not loaded at that time, i thought it was a standard ruby lib. To illustrate you can do in mdm master branch rails console: 
```
require 'webrick'
cookie = WEBrick::Cookie.new('test name', 'test value')
wp = Mdm::WebPage.create(cookie: cookie, web_site_id: 1, code: 'doge')
```
Then exit console, re-enter:
```
wp = Mdm::WebPage.last
```
And it will throw an error `undefined class/module WEBrick` reading it. Not sure what the proper solution is here.